src/dsl.ts

```diff
@@ -376,10 +404,10 @@ const identifier = lexeme(regex(/[a-zA-Z_][0-9a-zA-Z_+]*/));
 const path = lexeme(
   regex(/([a-zA-Z_][0-9a-zA-Z_+-]*\/)*[a-zA-Z_][0-9a-zA-Z_+-]*/),
 );
-const colon = lexeme(regex(/:/));
-const equal = lexeme(regex(/=/));
-const semicolon = lexeme(regex(/;/));
-const pipe = lexeme(regex(/\|/));
+const colon = lexeme(string(":"));
+const equal = lexeme(string("="));
+const semicolon = lexeme(string(";"));
+const pipe = lexeme(string("|"));
```

Comment:
I'm not sure why these were defined as regexes. They seem to work fine as `string`s.